packages/react-components/src/components/ReactionButton.tsx

```diff
@@ -123,6 +123,9 @@ export const ReactionButton = (props: ReactionButtonProps): JSX.Element => {
     <div style={reactionEmojiMenuStyles()}>
       {emojis.map((emoji, index) => {
         const resourceUrl = emojiResource.get(emoji);
+        const classname = mergeStyles(
+          emojiStyles(resourceUrl ? resourceUrl : '', isHoveredMap.get(emoji) ? 'running' : 'paused')
```

Comment:
I'll double check if we still need the isHovered map input here given we are just running the animation on hover

Reply:
(I should be able to move the transform into the hover css selector)

Reply:
Updated.